attributes/remi-debuginfo.rb

```diff
@@ -15,5 +15,5 @@
   default['yum']['remi-debuginfo']['baseurl'] = "http://rpms.remirepo.net/enterprise/#{node['platform_version'].to_i}/debug-remi/$basearch/"
   default['yum']['remi-debuginfo']['description'] = "Remi's RPM repository for Enterprise Linux #{node['platform_version'].to_i} - $basearch - debuginfo"
 else
-  raise "platform #{node['platform']} not recognised"
+  Chef::Log.warn("platform #{node['platform']} not recognised")
```

Comment:
I don't like having this either as it will fill up Debian users's warning log with stuff like:
```
[2021-11-08T13:06:28-08:00] WARN: platform ubuntu not recognised
[2021-11-08T13:06:28-08:00] WARN: platform ubuntu not recognised
[2021-11-08T13:06:28-08:00] WARN: platform ubuntu not recognised
[2021-11-08T13:06:28-08:00] WARN: platform ubuntu not recognised
[2021-11-08T13:06:28-08:00] WARN: platform ubuntu not recognised
[2021-11-08T13:06:28-08:00] WARN: platform ubuntu not recognised
[2021-11-08T13:06:28-08:00] WARN: platform ubuntu not recognised
[2021-11-08T13:06:28-08:00] WARN: platform ubuntu not recognised
[2021-11-08T13:06:28-08:00] WARN: platform ubuntu not recognised
[2021-11-08T13:06:28-08:00] WARN: platform ubuntu not recognised
[2021-11-08T13:06:28-08:00] WARN: platform ubuntu not recognised
[2021-11-08T13:06:28-08:00] WARN: platform ubuntu not recognised
[2021-11-08T13:06:28-08:00] WARN: platform ubuntu not recognised
[2021-11-08T13:06:28-08:00] WARN: platform ubuntu not recognised
[2021-11-08T13:06:28-08:00] WARN: platform ubuntu not recognised
```
For one, you have no idea where this warning is coming from and two it's repetitive. I'd recommend we do one of the following:

- Just remove it completely
- Switch it to debug and add `yum-remi-chef` to it so folks know where it's coming from.